data/hlint.yaml

```diff
@@ -739,6 +739,17 @@
     - package base
     rules:
     - hint: {lhs: "x /= []", rhs: not (null x), name: Use null}
+    - hint: {lhs: "[] /= x", rhs: not (null x), name: Use null}
```

Comment:
This one should be in teaching since the normal version is also in teaching. I'm happy with all the others being in the normal section.

Reply:
Okay, I've kept this one there, and moved all others next to their respective original hints.